neo/IO/Json/JNumber.cs

```diff
@@ -101,6 +101,7 @@ public override string ToString()
             return AsString();
         }
 
+        // TODO - WHO USES THIS?
```

Comment:
@erikzhang

Reply:
@vncoelho delete it, if this is being used, you will find out haha